docs/contributing-to-airbyte/resources/qa-checks.md

```diff
@@ -0,0 +1,81 @@
+
+## 📝 Metadata
+
+### Connectors must have valid metadata.yaml file
+*Applies to the following connector languages: java, low-code, python*
+
+Connectors must have a `metadata.yaml` file at the root of their directory. This file is used to build our connector registry. Its structure must follow our metadata schema. Field values are also validated. This is to ensure that all connectors have the required metadata fields and that the metadata is valid. More details in this [documentation](https://docs.airbyte.com/connector-development/connector-metadata-file).
+
+## 📦 Packaging
+
+### Connectors must use Poetry for dependency management
+*Applies to the following connector languages: python, low-code*
+
+Connectors must use [Poetry](https://python-poetry.org/) for dependency management. This is to ensure that all connectors use a dependency management tool which locks dependencies and ensures reproducible installs.
+### Connectors must be licensed under MIT or Elv2
+*Applies to the following connector languages: java, low-code, python*
+
+Connectors must be licensed under the MIT or Elv2 license. This is to ensure that all connectors are licensed under a permissive license. More details in our [License FAQ](https://docs.airbyte.com/developer-guides/licenses/license-faq).
+### Connector license in metadata.yaml and pyproject.toml file must match
+*Applies to the following connector languages: python, low-code*
+
+Connectors license in metadata.yaml and pyproject.toml file must match. This is to ensure that all connectors are consistently licensed
+### Connector version must follow Semantic Versioning
+*Applies to the following connector languages: java, low-code, python*
+
+Connector version must follow the Semantic Versioning scheme. This is to ensure that all connectors follow a consistent versioning scheme. Refer to our [Semantic Versioning for Connectors](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors) for more details.
+### Connector version in metadata.yaml and pyproject.toml file must match
+*Applies to the following connector languages: python, low-code*
+
+Connector version in metadata.yaml and pyproject.toml file must match. This is to ensure that connector release is consistent.
+### Python connectors must have PyPi publishing enabled
+*Applies to the following connector languages: python, low-code*
+
+Python connectors must have [PyPi](https://pypi.org/) publishing enabled in their `metadata.yaml` file. This is declared by setting `remoteRegistries.pypi.enabled` to `true` in metadata.yaml. This is to ensure that all connectors can be published to PyPi and can be used in `airbyte-lib`.
+
+## 💼 Assets
+
+### Connectors must have an icon
+*Applies to the following connector languages: java, low-code, python*
+
+Each connector must have an icon available in at the root of the connector code directory. It must be an SVG file named `icon.svg`.
```

Comment:
The certification process adds `and that the icon is square`. Is this validated? Should this be mentioned here even if we don't validate it?

Reply:
The square shape is not validated, we could validate it but this can come in a followup change. I'll update the check description to mention it has to be a square.